docs/quickstart.md

```diff
@@ -0,0 +1,411 @@
+
+```bash
+.
+├── components
+│   └── schemas
+│       ├── Error.yaml
+│       ├── Pet.yaml
+│       └── Pets.yaml
+├── paths
+│   ├── pets.yaml
+│   └── pets@{petId}.yaml
+└── openapi.yaml
+```
+
+With the `split` command, you can easily switch to a multi-file structure in a blink of an eye even if your definition file became extremely lengthy.
+
+### bundle
+
+With this command, you can merge standalone files back into a single definition file. `bundle` does the opposite of the [`split`](#split) command.
+
+Let's bundle the definition file that we split earlier.
+
+In your terminal, type the following:
+
+```bash
+$ openapi bundle openapi/openapi.yaml --output bundled.yaml
+```
+
+You should get the following response:
+
+```bash
+$ openapi bundle openapi/openapi.yaml --output bundled.yaml
+bundling openapi/openapi.yaml...
+📦 Created a bundle for openapi/openapi.yaml at bundled.yaml 46ms.
+```
+
+You should now see a new `bundled.yaml` file as a result of the `bundle` command completion. Open this file in your text editor or IDE and compare it with the linted definition file. You can see that both files are the same which means that the `bundle` command executed flawlessly.
+
+### preview-docs
+
+With this command, you can run a server to preview your OpenAPI definition file in real-time mode.
+
+In your terminal, type the following:
+
+```bash
+$ openapi preview-docs petstore.yaml 
+```
+
+You should get the following response:
+
+```bash
+$ openapi preview-docs petstore.yaml 
+Using Redoc community edition.
+Login with openapi-cli login or use an enterprise license key to preview with the premium docs.
+
+
+  🔎  Preview server running at http://127.0.0.1:8080
+
+Bundling...
+
+
+  👀  Watching petstore.yaml and all related resources for changes
+
+Created a bundle for petstore.yaml successfully
+```
+
+Navigate to `http://127.0.0.1:8080` and check that your definition file has been served successfully:
+
+![preview-docs](../images/preview-docs.png)
+
+This server support live changes - try to modify your definition file, these modifications will be previewed immediately.
+
+For example, change the version of your Swagger Petstore from `1.0.0` to `1.1.0`:
+
+```yaml petstore.yaml
+openapi: "3.0.0"
+info:
+  version: 1.1.0
+  ...
+```
+
+You should get the following response:
+
+```bash
+watch changed petstore.yaml
+watch changed /path-to-your-project/petstore.yaml
+
+Bundling...
+
+Created a bundle for petstore.yaml successfully
+GET /: 4.337ms
+GET /: 3.825ms
+GET /hot.js: 27.698ms
+GET /simplewebsocket.min.js: 42.654ms
+GET /: 6.77ms
+GET /hot.js: 1.639ms
+GET /simplewebsocket.min.js: 14.528ms
+GET /simplewebsocket.min.js: 1.47ms
+GET /hot.js: 1.826ms
+GET /openapi.json: 0.499ms
+GET /openapi.json: 2.293ms
+GET /openapi.json: 0.867ms
+```
+
+Navigate to `http://127.0.0.1:8080` and check the updated definition file:
+
+![updated-preview-docs](../images/updated-preview-docs.png)
+
+## Full list of commands
```

Comment:
As mentioned previously, it would be useful if the last section in the quickstart was called "Next steps" and then listed 2-3 specific actions the user can do. One of them can be checking out the full list of commands to learn more, yes. The other two could be something like learn more about OpenAPI, or learn how to add your API definitions to the registry... Or learning about the .redocly.yaml configuration file. The possibilities are almost endless :) but we should focus on the actions that bring the most value.